python-package/examples/RGF/comparison_RGF_and_RF_regressors_on_diabetes_dataset.py

```diff
@@ -24,20 +26,20 @@
 n_folds = 3
 
 rgf_scores = cross_val_score(rgf,
-                             boston.data,
-                             boston.target,
+                             diabetes.data,
+                             diabetes.target,
                              scoring=make_scorer(mean_squared_error),
                              cv=n_folds)
 rf_scores = cross_val_score(rf,
-                            boston.data,
-                            boston.target,
+                            diabetes.data,
+                            diabetes.target,
                             scoring=make_scorer(mean_squared_error),
                             cv=n_folds)
 
 rgf_score = sum(rgf_scores)/n_folds
 print('RGF Regressor MSE: {0:.5f}'.format(rgf_score))
-# >>>RGF Regressor MSE: 11.79409
+# >>> RGF Regressor MSE: 3377.46076
```

Comment:
MSE is large?
https://www.researchgate.net/figure/MSE-Convergence-of-used-algorithms-for-Diabetes-dataset_fig3_346454502

How about increse max_leaf and n_iter?

Reply:
@fukatani Thanks a lot for your feedback!

Yeah, I tried to increase those hyperparameters, of course. Unfortunately, increasing them makes results only worse.

I believe that this example is to show that RGF can outperform RF in some cases. And this goal is already achieved
```
RGF Regressor MSE: 3377.46076
Random Forest Regressor MSE: 3441.01988
```

Also, I have a great doubt that the article you've mentioned is referring the same dataset used in scikit-learn. MSE less than 0.55 for y scaled from 25 to 346 looks unbelievable.

To prove my words, here are some examples of MSE for this dataset over the GitHub:
https://github.com/Akshaykumarcp/ML-ensemble-learning/blob/b142d7d2ac32554fe8dba545bfcb0a2fb31e02da/0.2_generative_ensemble_methods/0.2.2_boosting_method/0.2.2.3_adaboost_scikit_regression.py#L37-L40
https://github.com/Mean518/kingminji/blob/5a3cd3ad7ace4e18fbbb07f1c8977b377e42e6b3/keras/keras80_diabetes_lstm.py#L61
https://github.com/Mean518/kingminji/blob/5a3cd3ad7ace4e18fbbb07f1c8977b377e42e6b3/keras/keras79_diabetes_dnn.py#L76
https://github.com/Mean518/kingminji/blob/5a3cd3ad7ace4e18fbbb07f1c8977b377e42e6b3/keras/keras81_diabetes_cnn.py#L79
https://github.com/kevinpiger/ML-100Days/blob/master/Day036-040/Day_040_lasso_ridge_regression.ipynb

Reply:
OK, now I'm sure these datasets are different:

> Source URL: https://www4.stat.ncsu.edu/~boos/var.select/diabetes.html
   https://scikit-learn.org/stable/datasets/toy_dataset.html#diabetes-dataset

Size: 11x442.


> The diabetes dataset containing multivariate characteristicsthat are extracted from the University of California machine learn-ing repository contained 768 specimens of adult females. This depicts chemical changes occurs from initial to peak stages in the female body that result in diabetes (Smith et al., 1988).

Reply:
Thx!
LGTM.

Sorry for very late response :pray:

Reply:
No problem at all! Thank you very much for the thoughtful review!